docs/_documentations/installoncloud.md

```diff
@@ -12,95 +12,47 @@ parent: root
+To setup a Codewind-ready install of Che, follow these instructions.
 
 
 # Table of Contents
 1. [Prerequisites](#prerequisites)
-2. [Installing Che with the operator](#installing-che-with-the-operator)
-3. [Setting up OKD and OpenShift](#setting-up-okd-and-openshift)
-4. [Setting up other Kubernetes](#setting-up-other-kubernetes)
-5. [After installing Che](#after-installing-che)
-6. [Creating the Codewind workspace](#creating-the-codewind-workspace)
-7. [Updating the version](#updating-the-version)
+2. [Installing Che with chectl](#installing-che-with-chectl)
+3. [Enable privileged and root containers to run](#enable-privileged-and-root-containers-to-run)
+4. [After installing Che](#after-installing-che)
+5. [Creating the Codewind workspace](#creating-the-codewind-workspace)
+6. [Updating the version](#updating-the-version)
 
 ## Prerequisites
 - Set up the PersistentVolume (PV) with either Network File System (NFS) or GlusterFS.
-  - For NFS, if running on IBM Cloud Private, follow these instructions:[NFS Storage Install Instructions](https://www.ibm.com/developerworks/community/blogs/fe25b4ef-ea6a-4d86-a629-6f87ccf4649e/entry/Working_with_storage?lang=en) and use 777 permissions for the folders.
+  - For NFS, we recommend 777 permissions for the exported folders and ownership of `nobody:nogroup`
   - You do not need to set up the PV for local Kube, such as Minikube, Minishift, Docker Desktop, and others.
-- Set up the ClusterRole.
-  - **Important:** If you are installing Che with the `deploy_che.sh` script for OpenShift, do this step *after* installing Che.
+- Ensure cluster can pull images from `docker.io/eclipse`
+  - Both Eclipse Che and Eclipse Codewind hosts their docker images on `docker.io/eclipse`. Ensure that your cluster can pull from that registry and does not have PodSecurityPolicies blocking it from accessing dockerhub.
+- Set up the ClusterRole for Codewind.
   1. Clone the [Codewind Che plug-in repository](https://github.com/eclipse/codewind-che-plugin)
   2. Enter the `cd` command to go to the `codewind-che-plugin` repository.
   3. Run the `kubectl apply -f setup/install_che/codewind-clusterrole.yaml` command to create a cluster role with the required permission.
   4. Next, run the `kubectl apply -f setup/install_che/codewind-rolebinding.yaml` command.
 
-## Installing Che with the operator
+## Installing Che with chectl
 
-For instructions on installing Che with the Che operator, please see the [Eclipse Che operator on Operator Hub](https://operatorhub.io/operator/eclipse-che).
+We recommend using the `chectl` CLI to install Eclipse Che for use with Codewind. For instructions on installing the `chectl` CLI tool, please see [Installing the chectl management tool](https://www.eclipse.org/che/docs/che-7/installing-the-chectl-management-tool/)
```

Comment:
- We try to avoid "recommend" and instead either simply state what the user needs to do or the reason behind the recommendation.
- For example, could we say, "The fastest way to install Eclipse Che for use with Codewind is to use the `chectl` CLI."?
- We generally also try to avoid "please" and just say, "see" :)